### PR TITLE
Roadmap for Ansible 6

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
To be discussed in https://github.com/ansible-community/community-topics/issues/56.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/roadmap/COLLECTIONS_6.rst

### CODE REVIEW

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst [hunk 1]
```diff
@@ -57,3 +57,10 @@ Ansible 6.x minor releases will occur approximately every three weeks if changes
 
 
 For more information, reach out on a mailing list or a chat channel - see :ref:`communication` for more details.
+
+Planned work
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/COLLECTIONS_6.rst:61]: Do we have GH Issues (or PRs) for this work, might be good to include links.

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst [hunk 2]
```diff
@@ -57,3 +57,10 @@ Ansible 6.x minor releases will occur approximately every three weeks if changes
...
+============
+
+* Remove compatibility code which prevents parallel install of Ansible 6 with Ansible 2.9 or ansible-base 2.10
+* Stop installing files we have no use for.
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/COLLECTIONS_6.rst]: Maybe give some examples?
```suggestion
* Stop installing files (such as X,Y, Z) we have no use for.
```

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst [hunk 3]
```diff
@@ -57,3 +57,10 @@ Ansible 6.x minor releases will occur approximately every three weeks if changes
...
+* Ship wheels (as ansible-core 2.13 will likely also do)
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/COLLECTIONS_6.rst]: Not sure what the correct full term is. Maybe add a note to say who/when you'd use these.

```suggestion
* Ship Pythno wheels (as ansible-core 2.13 will likely also do)
```

Comment (felixfontein) [reply docs/docsite/rst/roadmap/COLLECTIONS_6.rst]:
```suggestion
* Ship Python wheels (as ansible-core 2.13 will likely also do) to improve installation performance
```

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst (hunk 2 above)
Comment (felixfontein) [reply docs/docsite/rst/roadmap/COLLECTIONS_6.rst]:
```suggestion
* Stop installing files (such as tests and development artefacts like editor configs) we have no use for
```

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst (hunk 1 above)
Comment (felixfontein) [reply docs/docsite/rst/roadmap/COLLECTIONS_6.rst:61]: I've added a generic link to the community-topics issue (377e43d5e0cda574e2081839ef30dd390771cdd3). If we want to make this a precise list, we can probably only merge this shortly before Ansible 6 is ready. I'm even considering throwing out this session so we can first fix the timeframe before concentrating on details of what to do.

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst [hunk 4]
```diff
@@ -14,25 +14,25 @@ Release schedule
...
-:TBD: Release of Ansible-6.1.0 (bugfix + compatible features: every three weeks.)
+:2021-11-30: New Collections can be reviewed for inclusion in Ansible 6. Submit a request to include a new collection in this `GitHub Discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
+:2022-03-28: ansible-core feature freeze.
+:2022-04-11: Start of ansible-core 2.13 betas (biweekly, as needed).
+:2022-04-12: Ansible-6.0.0 alpha1 (roughly biweekly ``ansible`` alphas timed to coincide with ``ansible-core-2.13`` pre-releases).
+:2022-04-26: Last day for new collections to be submitted for inclusion in Ansible-6. Collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
+:2022-04-27: Community Meeting topic: List any new collection reviews which block release. List any backwards incompatible collection releases that beta1 should try to accommodate.
+:2022-05-02: First ansible-core release candidate, stable-2.13 branch created.
```
Comment (samccann) [inline docs/docsite/rst/roadmap/COLLECTIONS_6.rst]: core is pulling the branch earlier now. see - https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_13.html

Comment (felixfontein) [reply docs/docsite/rst/roadmap/COLLECTIONS_6.rst]: Right, I did wanted to update that, but I apparently didn't...

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst [hunk 5]
```diff
@@ -57,3 +57,12 @@ Ansible 6.x minor releases will occur approximately every three weeks if changes
...
 
 For more information, reach out on a mailing list or a chat channel - see :ref:`communication` for more details.
+
+Planned work
+============
+
+More details can be found in `the community-topics planning issue <https://github.com/ansible-community/community-topics/issues/56>`_.
+
+* Remove compatibility code which prevents parallel install of Ansible 6 with Ansible 2.9 or ansible-base 2.10
+* Stop installing files (such as tests and development artefacts like editor configs) we have no use for
```
Comment (samccann) [inline docs/docsite/rst/roadmap/COLLECTIONS_6.rst]:
```suggestion
* Stop installing files (such as tests and development artifacts like editor configs) we have no use for
```

**Comment by jillr** [comment]
LGTM - +1

File: docs/docsite/rst/roadmap/COLLECTIONS_6.rst (hunk 1 above)
Comment (gundalow) [reply docs/docsite/rst/roadmap/COLLECTIONS_6.rst:61]: Ok, this is good enough for the moment. Thanks for adding the extra details

**Review by gundalow — APPROVED** [review]
Thanks :)

**Comment by felixfontein** [comment]
**The proposed PR in the current form is being voted on.** Please vote in https://github.com/ansible-community/community-topics/issues/56. Thanks! The deadline for the vote is in one week.

**Comment by felixfontein** [comment]
This has been approved by the Ansible community steering committee, see https://github.com/ansible-community/community-topics/issues/56#issuecomment-1022466284 and the next comments.

**Review by samccann — APPROVED** [review]

**Comment by felixfontein** [comment]
@samccann @gundalow @jillr and everyone else who looked at this, thanks a lot!

**Pull request merged**